src/App.jsx

```diff
@@ -1,34 +1,37 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+// LIBRARY IMPORTS
+import React from 'react';
+import {
+	BrowserRouter as Router,
+	Routes,
+	Route,
+} from 'react-router-dom';
+
+// LOCAL IMPORTS
+import Card from './components/Card';
+import { Home, Layout, ProfessionalsPage } from './views';
 
 function App() {
-  const [count, setCount] = useState(0)
 
   return (
-    <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.jsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </>
+    <Router>
+      <Routes>
+        <Route
+          path="/"
+          element={<Layout />}
+        >
+          <Route
+						index
+						element={
+							<Home />
+						}
+					/>
+          <Route
+						path="/professionals"
+						element={<ProfessionalsPage />}
+					/>
+        </Route>
+      </Routes>
```

Comment:
These 'sub-routes' under the Layout route specify actual paths to the home page and professionals page